src/v2/view-builder/utils/LinksUtil.js

```diff
@@ -113,9 +113,26 @@ const getSignOutLink = (settings) => {
   }
 };
 
+const getDefaultBackSignInLink = (settings, appState) => {
+  const messagesObjs = appState?.get('messages');
+  if (messagesObjs && Array.isArray(messagesObjs.value)
+    && messagesObjs.value.some(messagesObj => messagesObj.i18n?.key === 'idx.return.link.expired')) {
```

Comment:
Instead of using this check, i'd recommend using a separate remediation object for this. Perhaps work with backend on it? Any thoughts @haishengwu-okta ?

Reply:
It's actually good idea but not sure if it might be overkill to have separate remediation object for this.
Worth to check with IdX team though @VadymYaremchuk-okta

Reply:
According to our decision ```We are going to revisit the use cases for Embedded Deployment Models will be explored and speced later``` we may keep the fix simple for now and then update it with remediation object later if we need to.

Btw, are we expecting by design to have remediation object in terminal state? Since I supposed that remediation can be present only while we are not in terminal state.